src/assets/styles/global/colors.scss

```diff
@@ -1,5 +1,11 @@
 $p-primary-button-background: #e1b382;
 $p-error-color: #ff0000;
 $p-color-grey: #c3c3c3;
+$p-color-light-grey: #EEEEEE;
 $p-color-white: #ffffff;
 $p-color-black: #000000;
+$p-color-17: #171717;
```

Comment:
@kovacevicmilica544 I'm not sure who introduced those, but same comment apply here, @dejanmijic  look at her PR so you will get familier with it.

Also if you both agree, we can add comment above those colors in which context they have been used?